### PR TITLE
feat(demo): update Limit card page

### PR DESCRIPTION
**Pull request by aiheon**
### Proposed Changes

https://coveord.atlassian.net/browse/UITOOL-533

### Potential Breaking Changes

none

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/website/src/pages/layout/Limit.tsx [hunk 1]
```diff
@@ -1,86 +1,69 @@
...
-                    <Limit id="8" title={title} usage={17} isLimitEditable />
-                </div>
-            </div>
-        </div>
-    </PlasmaComponent>
+        componentSourcePath="/limit/Limit.tsx"
+        title="Limit card"
+        section="Layout"
+        description="A limit card displays the limit and usage of a resource. It includes a bar illustrating the usage against the limit."
+        thumbnail="placeholder"
```
Comment (gdostie) [inline packages/website/src/pages/layout/Limit.tsx:58]: this prop defaults to placeholder, no need to specify it

```suggestion
```

File: packages/react/src/components/limit/Limit.tsx [hunk 2]
```diff
@@ -11,15 +11,47 @@ import {InputSelectors} from '../input/InputSelectors';
...
+    /**
+     * Display the current usage under the Usage label
+     */
     usage?: number;
+    /**
+     * Set the limit value of the Limit card. When the limit is reached, the progress bar is turning red color.
+     */
     limit?: number;
+    /**
+     * Whether or not the limit value is editable directly in the Limit card
```
Comment (gdostie) [inline packages/react/src/components/limit/Limit.tsx:41]: I think we should omit the `or not` everywhere https://www.grammarphobia.com/blog/2013/03/whether-or-not.html

```suggestion
     * Whether the limit value is editable directly in the Limit card
```

**Review by gdostie — APPROVED** [review]

**Review by GermainBergeron — COMMENTED** [review]
There's an when you edit the issue editable limit, it updates the previous exemple too. Are we missing some ids?

https://user-images.githubusercontent.com/260007/158823308-376d8280-820c-4c2b-aa7d-0027729b039b.mov

File: packages/website/src/pages/layout/Limit.tsx [hunk 3]
```diff
@@ -1,86 +1,68 @@
...
+    import * as React from 'react';
+    import {Limit} from '@coveord/plasma-react';
+
+    export default () => (
+        <Limit title={'Hey'} usage={82} isHistoryIncluded limit={100} onHistoryIconClick={() => alert('Patate!')}
+        />
+    );
+`;
+
+const withcustomValue = `
```
Comment (GermainBergeron) [inline packages/website/src/pages/layout/Limit.tsx:33]:
```suggestion
const withCustomValue = `
```

File: packages/website/src/pages/layout/Limit.tsx [hunk 4]
```diff
@@ -1,86 +1,68 @@
...
-    </PlasmaComponent>
+        componentSourcePath="/limit/Limit.tsx"
+        title="Limit card"
+        section="Layout"
+        description="A limit card displays the limit and usage of a resource. It includes a bar illustrating the usage against the limit."
+        code={code}
+        examples={{
+            withGoal: {code: withGoal, title: 'With goal to reach'},
+            withHistory: {code: withHistory, title: 'With History'},
+            withcustomValue: {code: withcustomValue, title: 'With custom value'},
```
Comment (GermainBergeron) [inline packages/website/src/pages/layout/Limit.tsx:62]:
```suggestion
            withCustomValue: {code: withCustomValue, title: 'With custom value'},
```

File: packages/react/src/components/limit/Limit.tsx [hunk 5]
```diff
@@ -11,15 +11,47 @@ import {InputSelectors} from '../input/InputSelectors';
...
+    /**
+     * Whether the limit value is editable directly in the Limit card
+     */
     isLimitEditable?: boolean;
+    /**
+     * Whether or not the limit value is a goal. When the goal is achived, the progress bar stay teal color instead of turning red
+     */
     isLimitTheGoalToReach?: boolean;
+    /**
+     * Whether or not the history icons is rendered
```
Comment (gdostie) [inline packages/react/src/components/limit/Limit.tsx:49]:
```suggestion
     * Whether the history icons is rendered
```

**Comment by aiheon** [comment]
> There's an when you edit the issue editable limit, it updates the previous exemple too. Are we missing some ids?
> 
>  Screen.Recording.2022-03-17.at.9.30.09.AM.mov

Fixed! :smile:

**Review by GermainBergeron — APPROVED** [review]

**Pull request merged**